osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/DlmsConnectionManager.java

```diff
@@ -146,8 +147,8 @@ public void reconnect() throws OsgpException {
     public void close() {
         try {
             this.dlmsConnection.close();
-        } catch (final IOException e) {
-            LOGGER.warn("Failure while trying to close a DLMS connection", e);
+        } catch (final Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1166)